[2021-12-17 04:44:54,775] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: marketvol.task_3 2021-12-17T04:44:37.911384+00:00 [queued]>
[2021-12-17 04:44:54,802] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: marketvol.task_3 2021-12-17T04:44:37.911384+00:00 [queued]>
[2021-12-17 04:44:54,803] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-12-17 04:44:54,804] {taskinstance.py:1043} INFO - Starting attempt 1 of 3
[2021-12-17 04:44:54,805] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-12-17 04:44:54,887] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): task_3> on 2021-12-17T04:44:37.911384+00:00
[2021-12-17 04:44:54,892] {standard_task_runner.py:52} INFO - Started process 1498 to run task
[2021-12-17 04:44:54,899] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'marketvol', 'task_3', '2021-12-17T04:44:37.911384+00:00', '--job-id', '5', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmplcck812l', '--error-file', '/tmp/tmpw96ia3u6']
[2021-12-17 04:44:54,903] {standard_task_runner.py:77} INFO - Job 5: Subtask task_3
[2021-12-17 04:44:55,038] {logging_mixin.py:104} INFO - Running <TaskInstance: marketvol.task_3 2021-12-17T04:44:37.911384+00:00 [running]> on host d129b14a7a78
[2021-12-17 04:44:55,138] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=marketvol
AIRFLOW_CTX_TASK_ID=task_3
AIRFLOW_CTX_EXECUTION_DATE=2021-12-17T04:44:37.911384+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-17T04:44:37.911384+00:00
[2021-12-17 04:44:55,140] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-12-17 04:44:55,142] {bash.py:158} INFO - Running command: mv $AIRFLOW_HOME/AAPL_data.csv $AIRFLOW_HOME/tmp/data/2021-12-16
[2021-12-17 04:44:55,152] {bash.py:169} INFO - Output:
[2021-12-17 04:44:55,163] {bash.py:177} INFO - Command exited with return code 0
[2021-12-17 04:44:55,234] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=marketvol, task_id=task_3, execution_date=20211217T044437, start_date=20211217T044454, end_date=20211217T044455
[2021-12-17 04:44:55,359] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-12-17 04:44:55,410] {local_task_job.py:146} INFO - Task exited with return code 0
